src/components/Newsletter/Newsletter.tsx

```diff
@@ -0,0 +1,156 @@
+import { useEffect } from "react"
+import { styled } from "styled-components"
+import { breakpoints, theme } from "../../utils/theme"
+
+const Newsletter = () => {
+  useEffect(() => {
+    const script = document.createElement("script")
+    script.src = "https://subscribe-forms.beehiiv.com/embed.js"
+    script.async = true
+    document.body.appendChild(script)
+
+    return () => {
+      document.body.removeChild(script) // Limpieza al desmontar el componente
+    }
+  }, [])
+  return (
+    <NewsletterContainer>
+      <iframe
+        src="https://subscribe-forms.beehiiv.com/a3c4e913-9f95-435e-b12f-d2f0271dd0ad"
```

Comment:
wdyt about using: `?slim=true` to show the message inline instead of loading the JS file?